modules/nf-core/motus/downloaddb/tests/main.nf.test

```diff
@@ -28,4 +28,23 @@ nextflow_process {
 
     }
 
+    test("Should download mOTUs database - stub") {
+
```

Comment:
```suggestion
options '-stub'
```

Reply:
Updated the PR! I realised it late that I forgot the `stub` options :D